content/en/llm_observability/setup/auto_instrumentation.md

```diff
@@ -16,12 +16,12 @@ further_reading:
 Datadog's [LLM Observability Python SDK][16] provides integrations that automatically trace and annotate calls to LLM frameworks and libraries. Without changing your code, you can get out-of-the-box traces and observability for calls that your LLM application makes to the following frameworks:
 
 
-| Framework                               | Supported Versions |
-|-----------------------------------------|--------------------|
-| [OpenAI](#openai)                       | >= 0.26.5          |
-| [Langchain](#langchain)                 | >= 0.0.192         |
-| [AWS Bedrock](#aws-bedrock)             | >= 1.31.57         |
-| [Anthropic](#anthropic)                 | >= 0.28.0          |
+| Framework                               | Supported Versions | Tracer Version    |
+|-----------------------------------------|--------------------|-------------------|
+| [OpenAI](#openai)                       | >= 0.26.5          | >= 2.9.0          |
+| [Langchain](#langchain)                 | >= 0.0.192         | >= 2.9.0          |
+| [AWS Bedrock](#aws-bedrock)             | >= 1.31.57         | >= 2.9.0          |
```

Comment:
```suggestion
| [Amazon Bedrock](#amazon-bedrock)             | >= 1.31.57         | >= 2.9.0          |
```
Looks like Amazon calls this "Amazon Bedrock" (link)[https://aws.amazon.com/bedrock/?sec=aiapps&pos=2]. Would you mind changing it on line 85 as well?

Reply:
ah yes, can do! I was wondering what the fix to the check would be 😂

Reply:
fixed, we also referenced `AWS` a couple of other (non-link) places on this page that I fixed as well

Reply:
Yeah, usually it gives a recommendation 😅